js/pro/kraken.js

```diff
@@ -1004,7 +1004,7 @@ module.exports = class kraken extends krakenRest {
         if (rawTrades !== undefined) {
             trades = this.parseTrades (rawTrades, market, undefined, undefined, { 'order': id });
         }
-        const stopPrice = this.safeString (order, 'stopprice');
+        const stopPrice = this.safeNumber (order, 'stopprice');
```

Comment:
I thought `safeOrder` expected unparsed strings?

Reply:
@samgermain it does for most values, `stopPrice` is not one of them yet so we were sending a stringified number to the user